extensions/wikia/MercuryApi/tests/MercuryApiModelTest.php

```diff
@@ -12,19 +12,21 @@ public function setUp() {
 	 */
 	private function setupGlobals() {
 		$this->mockGlobalVariable('wgLoadAdsInHead', true);
+		$this->mockGlobalVariable('wgAdDriverBottomLeaderboardImpressionCapping', false);
```

Comment:
You're probably right :japanese_ogre: Changing it will probably require lots of changes in `MercuryApiModelTest.php` and since the default value is set in `DefaultSettings.php` it might not make sense to do the changes.